cinema/tests/test_movies.py

```diff
@@ -0,0 +1,160 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from cinema.models import Actor, Movie, Genre
+from cinema.serializers import MovieListSerializer, MovieDetailSerializer
+
+
+class MovieTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user = get_user_model().objects.create_user(
+            email="test@1234test", password="123123123"
+        )
+        self.superuser = get_user_model().objects.create_superuser(
+            email="test3523@test", password="admin12345"
+        )
+        for i in range(0, 11):
+            actor = Actor.objects.create(first_name=f"1{i}", last_name=f"2{i}")
+            genre = Genre.objects.create(name=f"10{i}")
+            movie = Movie.objects.create(
+                title=f"Sample{i}", description="Sampleee", duration=i * 10
+            )
+            movie.actors.add(actor)
+            movie.genres.add(genre)
+        Movie.objects.get(id=2).genres.add(Genre.objects.get(id=2))
+        Movie.objects.get(id=2).actors.add(Actor.objects.get(id=2))
```

Comment:
Ensure that the IDs used in `Movie.objects.get(id=2)` and similar calls exist in the database. If the database is reset between tests, these IDs might not be valid.